src/us_reverse_geo/SmartyResponse.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * A candidate is a possible match for an address that was submitted.<br>
+ *     A lookup can have multiple candidates if the address was ambiguous, and<br>
+ *     the maxCandidates field is set higher than 1.
+ *
+ * @see "https://smartystreets.com/docs/cloud/us-street-api#root"
+ */
+class SmartyResponse {
```

Comment:
This is only being used in the test file. Is that your intention?

Reply:
I'm still trying to get a grasp on the loosely typed nature of JavaScript. The purpose of this is to have the fields defined when unwrapping the JSON response, is this necessary?

Reply:
Necessary, no. If you're only going to be mapping the results to results and not defining the other fields then I don't think it's necessary here.

Also I think if you're going to define the results in a SmartyResponse for the tests, I think you should do that for the src as well. Most importantly though, I think it is important to be consistent across the SDK with the different APIs so using this here is kinda breaking that consistency. It would be better to use the `Result` class that you defined for the US Reverse Geo API instead (see `src/us_reverse_geo/Result.js`)